tests/test_gibbs.py

```diff
@@ -132,6 +136,60 @@ def test_horseshoe_nbinom(srng):
     assert np.all(lmbda > 0)
 
 
+def test_R_r(srng):
+    F = F_matrix_construct(50)
+    r_ = srng.gamma(1, 1)
+    R_r(F, r_, 4)
```

Comment:
This looks like an unfinished test. We definitely need a test for these components (i.e. everything involved in `sampling_for_li`, and that function itself), though.